tests/test_providers.py

```diff
@@ -40,7 +40,7 @@ def test_googleai():
 
 #
 #
-test_googleai()
+# test_googleai()
```

Comment:
Can you add a test for the model and embedding listing in `test_providers.py`?

Reply:
ack, left ticket tracking this